pkg/proc/native/ptrace_linux_amd64.go

```diff
@@ -39,3 +39,31 @@ func PtraceGetRegset(tid int) (regset linutil.AMD64Xstate, err error) {
 	err = linutil.AMD64XstateRead(regset.Xsave, false, &regset)
 	return
 }
+
+// ProcessVmRead calls process_vm_readv
+func ProcessVmRead(tid int, addr uintptr, data []byte) (int, error) {
```

Comment:
Make a file called ptrace_linux_64bit.go and add this at the top:

```
// +build linux,amd64 linux,arm64
```

and put ProcessVmRead, ProcessVmWrite inside it.

Reply:
Ok, I will do it.